src/Sarif.Converters/FortifyFprConverter.cs

```diff
@@ -488,6 +489,18 @@ private void ParseVulnerability(IResultLogWriter output)
                 {
                     ParseLocationsFromTraces(result);
                 }
+                else if (AtStartOfNonEmpty(_strings.DefaultSeverity))
+                {
+                    rule.DefaultConfiguration.SetProperty("Deprecated"+ _strings.DefaultSeverity, _reader.ReadElementContentAsString());
```

Comment:
>recated"+  [](start = 62, length = 10)

NIT: missing space before the plus sign here and on line 498,

Reply:
Actually, the requirement was to write "InstanceSeverity" as "DepricatedInstanceSeverity" (without any spaces)

Reply:
I may have been unclear, in your code, there is a space missing before the '+' operator, just a style note. I am not commenting here on the name of the property.

---
In reply to: [315399017](https://github.com/microsoft/sarif-sdk/pull/1650#discussion_r315399017) [](ancestors = 315399017)